diff --git a/node_modules/@clerk/clerk-react/dist/index.js b/node_modules/@clerk/clerk-react/dist/index.js
index c86ea39..f500a67 100644
--- a/node_modules/@clerk/clerk-react/dist/index.js
+++ b/node_modules/@clerk/clerk-react/dist/index.js
@@ -93,7 +93,7 @@ var import_loadClerkJsScript2 = require("@clerk/shared/loadClerkJsScript");
 var import_error = require("@clerk/shared/error");
 var errorThrower = (0, import_error.buildErrorThrower)({ packageName: "@clerk/clerk-react" });
 function setErrorThrowerOptions(options) {
-  errorThrower.setMessages(options).setPackageName(options);
+  errorThrower.setMessages(options).setPackageName(options.packageName); //changed
 }
 
 // src/components/uiComponents.tsx
